com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -20,9 +21,14 @@ public enum RenderingMode
     /// </summary>
     public sealed class ForwardRenderer : ScriptableRenderer
     {
-        static readonly int k_DepthStencilBufferBits = 32;
-        static readonly string k_CreateCameraTextures = "Create Camera Texture";
-        private static readonly ProfilingSampler m_ProfilingSampler = new ProfilingSampler(k_CreateCameraTextures);
+        const int k_DepthStencilBufferBits = 32;
+
+        private static class Profiling
+        {
+            // NOTE: field names start with lowercase to avoid shadowing method names.
```

Comment:
I thought lower case  applied to properties only, not fields. The comment is there because if these were in upper case they would shadow the method names in this scope and we would need to write FowardRenderer.method to reference the method itself can get protection against renaming in the nameof().
I'll fix the constants.

Reply:
It also applies to public fields :) Since it's in a sub class, it does not shadow the method anyway.

Reply:
I can remove the comment. I had the shadowing bug already :), but that is pointless now.

Reply:
Removed